invary_runtime_integrity/README.md

```diff
@@ -0,0 +1,44 @@
+# Invary Runtime Integrity
+
+Visualize the Runtime Integrity of your Invary managed operating systems.
+
+## Overview
+
+Invary validates the Runtime Integrity of operating systems and detects rootkits that can deceive other systems. This integration comes with an out-of-the-box dashboard that visualizes your Runtime Integrity, providing confidence in your runtime security or spotlighting endpoints that lack integrity and may be compromised. The Invary dashboard also provides insights into your operating system inventory at runtime, including distributions and kernel versions in use. 
+
+
+This integration uses the [Invary API][1].
+
+## Setup
+
+- Once this integration has been uninstalled, any previous authorizations are revoked. 
+- Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys page][3].
+
+### Installation
+
+The Invary integration allows you to forward details about your endpoint and appraisal from the Invary SaaS platform to your Datadog instance. No additional installation is needed on your server.
+
+### Configuration
+
+1. Complete the OAuth Authorization flow allowing Invary to communicate with your Datadog instance.
+2. Review the "Invary Runtime Integrity" dashboard for an aggregate look at your Runtime Integrity.
+
+### Validation
+
+1. Review the "Invary Runtime Integrity" dashboard for timely and expected appraisal information.
+2. Query your logs with the 'source:invary' base query.
+
+## Data Collected
+
```

Comment:
```suggestion

### Metrics

The Invary Runtime Integrity integration does not include any service checks.

### Service Checks

The Invary Runtime Integrity integration does not include any service checks.

### Events

The Invary Runtime Integrity integration does not include any events.
```

Reply:
@alai97 Thanks for the feedback. Do we want it to read "does not include any metrics." under the ###Metrics heading?